packages/sdk/package.json

```diff
@@ -113,6 +113,7 @@
     "bn.js": "^5.2.1",
     "bs58": "^5.0.0",
     "cbor-web": "^8.1.0",
+    "cbor-x": "^1.4.1",
```

Comment:
cbor-x looks pretty good, we can remove cbor-web then right?

Reply:
done in latest